### DISCUSSION TITLE
Updating multiple repositories with Scala Steward and GitHub Actions

### DISCUSSION THREAD

**Discussion by giscus** [alejandrohdezma/alejandrohdezma#148] [Blog]
This discussion contains the comments and reactions for [this blog post](https://alejandrohdezma.com/blog/updating-multiple-repositories-with-scala-steward-and-github-actions).

**Comment by pepegar** [comment]
This is very cool, thanks!

**Comment by gutiory** [comment]
Super useful article!. Thanks a lot for sharing with us :heart:.
